utils/credentialsUtils.js

```diff
@@ -0,0 +1,137 @@
+import { v4 as uuidv4 } from 'uuid';
+import { toast } from 'sonner';
+import { createBankIdCredential } from '../_credentials/quotient';
+import { createCreditScoreCredential } from '../_credentials/equinet';
+import { createBiometricsCredential } from '../_credentials/forsur';
+import { postRequest } from './request';
+
+const dockUrl = 'https://api-testnet.dock.io';
```

Comment:
We should use an env variable here

Reply:
```suggestion
const dockUrl = process.env.DOCK_API_URL;
```

Reply:
This change is ok but please commit them and mark the conversation as resolved after so it's easier to track.

Reply:
Fixed all the dockUrl constans to use the .env